doc/source/timeseries.rst

```diff
@@ -557,6 +557,40 @@ We are stopping on the included end-point as it is part of the index
    dft2 = dft2.swaplevel(0, 1).sort_index()
    dft2.loc[idx[:, '2013-01-05'], :]
 
+
```

Comment:
note that this behavior works with Timestamps or strings

Reply:
actually also `datetime.datetime` as `pd.Timestamp` can also digest that. Should it be a note like I did now?